src/helpers/highlight.ts

```diff
@@ -15,4 +15,20 @@ export async function storeMsg(id, ipfs, address, version, timestamp, space, typ
       receipt
     }
   ]);
+  if (result.insertId) {
+    await sequencerDB.queryAsync(query, [
+      {
+        id,
+        mci: result.insertId,
+        ipfs,
+        address,
+        version,
+        timestamp,
+        space,
+        type,
+        sig,
+        receipt
+      }
+    ]);
+  }
```

Comment:
I assume this PR is to make the messages save in both DB at the same time, and another PR will remove the old insert into the hub db ?

Reply:
Correct, to make sure there is 0 downtime.